velox/docs/functions/spark/string.rst

```diff
@@ -188,6 +188,13 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
         SELECT rtrim('kr', 'spark'); -- "spa"
 
+.. spark:function:: soundex(string) -> string
+
+    Returns Soundex code of the string. If first character of ``string`` is not
```

Comment:
nit: Perhaps, turn Soundex into a link to https://en.wikipedia.org/wiki/Soundex to help readers who are not familiar with this algorithm.